### PR TITLE
Stronger Improvised Firebombs

### PR DESCRIPTION
**Pull request by yoyobatty**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Improved Firebombs
add: Slightly stronger IED fire bomb you can craft with 100 fuel instead of 50.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
Change the changelog to include your name instead of "Optional Name Here"

File: code/game/objects/items/grenades/ghettobomb.dm [hunk 1]
```diff
@@ -19,6 +19,10 @@
 	var/range = 3
 	var/list/times
 
+/obj/item/grenade/iedcasing/upgrade
```
Comment (ghost) [inline code/game/objects/items/grenades/ghettobomb.dm]: not moduralised!
take all the changes in ghettobomb.dm
make a new file as so
hippiestation/code/game/objects/items/grenandes/ghettobomb.dm
past all the new code you made here.

there should be no changes in this ghettobomb.dm

**Review by ghost — DISMISSED** [review]

**Review by ghost — CHANGES_REQUESTED** [review]
Changelog will print the change log author as IED grenades when it should be YoYoBatty

File: hippiestation/code/game/objects/items/grenades/ghettobombupgrade.dm [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+/obj/item/grenade/iedcasing/upgrade
+	name = "improved improvised firebomb"
+	desc = "A stringer, improvised incendiary device."
```
Comment (GuyonBroadway) [inline hippiestation/code/game/objects/items/grenades/ghettobombupgrade.dm:4]: Did you not mean to put "Stronger" instead of "Stringer"

**Review by GuyonBroadway — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Yes please

**Comment by ghost** [comment]
I don't really like the idea of a bigger explosion since blowing things up is already nasty.
Why not make it like 50 Fuel, 50 ethanol for a bigger fire based explosion? Perhaps a little less boom and more fire. Otherwise this is a straight upgrade to the normal version and it won't ever be crafted again.

**Comment by tortellinitony** [comment]
Why? We really don't need this...

**Comment by Swagile** [comment]
what mcbawg said

**Comment by praisenarsie** [comment]
what the pasta man said

**Comment by KayeArray** [comment]
-1 from me because it's a boring way to buff greytide imo

**Comment by dreamcrab** [comment]
i like the idea of making it have more fire and burning for longer, but big explosions for greytide seems gay

PLEASE REVISE

**Comment by KayeArray** [comment]
I actually have my own idea for a revised reagent fireball system at some point.
Judging  by the negative input I will be closing this.

**Pull request closed without merging**